packages/wrangler/src/metrics/send-event.ts

```diff
@@ -71,7 +71,8 @@ export type EventNames =
 	| "list worker versions"
 	| "view versioned deployment"
 	| "view latest versioned deployment"
-	| "list versioned deployments";
+	| "list versioned deployments"
+	| "pages download config";
```

Comment:
looking at these events, the naming convention for pages seems to be `<action> pages ...`. So perhaps we can keep that convention here to and use `download pages config` instead?

Reply:
Yes that is correct @CarmenPopoviciu  - sparrow metrics events have a specific convention.

Reply:
Changed